New feature flag: allow_error_cb_on_chord_header - allowing setting an error callback on chord header (#7712)
celery/celery: docs/userguide/calling.rst

callbacks will be called with the return value of the parent task
as a partial argument.

> In case of a chord, error handling may have multiple handling strategies.

Reviewer: ```suggestion
In the case of a chord, we can handle errors using multiple handling strategies.
```
